packages/components/src/toolbar-button/index.js

```diff
@@ -48,8 +48,9 @@ function ToolbarButton( {
 					isPressed={ props.isActive }
 					disabled={ props.isDisabled }
 					{ ...extraProps }
-				/>
-				{ children }
+				>
+					{ children }
```

Comment:
@youknowriad Is it possible that this now overrides what's being passed in `extraProps.children`? 
If you [passed a button text to the toolbar](https://github.com/Automattic/jetpack/blob/master/extensions/blocks/podcast-player/edit.js#L242-L244) previous to 7.9, that button now appears empty: https://github.com/Automattic/jetpack/issues/15459

Reply:
For me the change here is not to avoid passing "children" but moving the children position inside the button which makes more sense IMO

Reply:
That said, I'm still wondering why it regressed, the children should still show up for toolbar buttons? 🤔

Reply:
It looks like the "inner" `children` supersedes the `children` that is passed as a prop?

We fixed it in https://github.com/Automattic/jetpack/pull/15508 through emulating how `MediaReplaceFlow` does it, so don't feel inclined to change anything on behalf of this. Awareness might be enough in case others run into this at some point

Reply:
We can also switch the priorities of the passed children (give the extraProps more priority), it is also weird to have the same prop in two different places, we should consolidate maybe.  I wonder where the "extraProp" prop is passed and in which case the children should be extracted and just passed as regular "children" prop.